src/boot/is_debug.c

```diff
@@ -2,14 +2,13 @@
 
 OSPiHandle* sISVHandle; // official name : is_Handle
 
-#define gISVDbgPrnAdrs ((ISVDbg*)0xb3ff0000)
-#define ASCII_TO_U32(a, b, c, d) ((u32)((a << 24) | (b << 16) | (c << 8) | (d << 0)))
+#define gISVDbgPrnAdrs ((ISVDbg*)0xB3FF0000)
 
 void isPrintfInit(void) {
     sISVHandle = osCartRomInit();
     osEPiWriteIo(sISVHandle, (u32)&gISVDbgPrnAdrs->put, 0);
     osEPiWriteIo(sISVHandle, (u32)&gISVDbgPrnAdrs->get, 0);
-    osEPiWriteIo(sISVHandle, (u32)&gISVDbgPrnAdrs->magic, ASCII_TO_U32('I', 'S', '6', '4'));
+    osEPiWriteIo(sISVHandle, (u32)&gISVDbgPrnAdrs->magic, 'IS64');
```

Comment:
I don't know how I feel about actually using multichars. It raises a gcc warning and it feels a bit like using an unsupported part of the C language (iirc it's implementation defined at best).

Reply:
For some reason I got the impression this was well defined but yeah it's implementation-defined, reverting